website_sale_checkout_skip_payment/models/website.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Sergio Teruel <sergio.teruel@tecnativa.com>
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import api, fields, models
+from odoo.http import request
+
+
+class Website(models.Model):
+    _inherit = "website"
+
+    checkout_skip_payment = fields.Boolean(
+        compute='_compute_checkout_skip_payment')
+
+    @api.multi
+    def _compute_checkout_skip_payment(self):
+        for rec in self:
+            if request.session.uid:
```

Comment:
just checking: is 100% that this is going to be recomputed for each user?
I'd rather use a std model method to get it not cached.

Reply:
@simahawk just tested with 2 different users on 2 different browsers: it works